net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
Consider keeping the copyright to yourself if you wrote this file.  Or just append your name there if you made significant changes to it ;)